manage_breast_screening/core/jinja2/django_form_helpers.jinja

```diff
@@ -59,3 +60,30 @@
 {% macro app_checkboxes(form_attribute, legend=None, hint=None) %}
   {{ _render(form_attribute=form_attribute, legend=legend, hint=hint, component=checkboxes_component) }}
 {% endmacro %}
+
+{% macro app_select(form_attribute, label=None, hint=None) %}
```

Comment:
I removed the usage of this - but thought it was worth keeping in for future forms. I'm hoping to work towards a kind of form builder that

- works with all the form inputs in the design system
- still allows forms to be defined declaratively using django.forms